test/fixtures/malformed/web-frame.md

```diff
@@ -109,6 +109,43 @@ this limitation.
 
 ### `webFrame.getResourceUsage()`
 
+Returns `Object`:
```

Comment:
This fixture is not "malformed", is it? It should probably go in a different fixture file/directory that is not named as such. Or you could change the vendored `electron/electron` submodule to point at a branch that has this new return object in its docs. When that lands in master we can change the submodule back to master. That's been my process as I make changes to the docs that are not yet in master.